server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileServicesImpl.java

```diff
@@ -23,18 +20,9 @@ public MemberProfile getById(UUID id) {
     }
 
     @Override
-    public Set<MemberProfile> findByValues(String name, String role, UUID pdlId) {
-        Set<MemberProfile> foundProfiles = new HashSet<>(memberProfileRepository.findAll());
-        if (!StringUtils.isEmpty(name)) {
-            foundProfiles.retainAll(memberProfileRepository.findByName(name));
-        }
-        if (!StringUtils.isEmpty(role)) {
-            foundProfiles.retainAll(memberProfileRepository.findByRole(role));
-        }
-        if (pdlId != null) {
-            foundProfiles.retainAll(memberProfileRepository.findByPdlId(pdlId));
-        }
-        return foundProfiles;
+    public List<MemberProfile> findByValues(String name, String role, UUID pdlId) {
```

Comment:
Switch pdlId to be a String, and pass that value right in.

Reply:
This will result in some refactoring of the tests.

Reply:
+1